src/runtime/components/forms/Input.vue

```diff
@@ -162,6 +162,10 @@ export default defineComponent({
     modelModifiers: {
       type: Object as PropType<{ trim?: boolean, lazy?: boolean, number?: boolean }>,
       default: () => ({})
+    },
+    eagerValidation: {
+      type: Boolean,
+      default: false
     }
```

Comment:
I think it would be better to set this parameter on the `FormGroup` component, since it already handles most of the validation logic. The prop can be added [here](https://github.com/nuxt/ui/blob/dev/src/runtime/components/forms/FormGroup.vue#L113) to pass it down into the composable.

Reply:
Great point! Resolved with latest commit